src/styles/page.css

```
@@ -0,0 +1,19 @@
+@import url(./card.css);
+@import url(./reset.css);
+@import url(./stars.css);
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-weight: 500;
+}
+
+:root {
+  --color-blue: #00acdc;
+  --color-white: #fff;
+  --dark: #060b35;
+  --grey: #616070;
```

dark and grey are not obvious names. Use color prefix
```suggestion
  --color-dark: #060b35;
  --color-grey: #616070;
```